File: src/Exercicio2.java
public class Exercicio2 {
    static void Fibonacci(int N) {
        int num1 = 0, num2 = 1;
        int contador = 0;

        while (contador < N) {
            System.out.print(num1 + " ");

            //Verificar sé o número de entrada faz parte da sequência de Fibonacci
            if (N == num1){
                System.out.printf("(%d)\n\nNúmero %d PERTENCE a sequência de Fibonacci!\n", num1, num1);
                break;
            }
            else if (num1 > N) {
                System.out.printf("\n\nO número %d NÃO pertence a sequência de Fibonacci\n", num1);
                break;
            }

            int num3 = num2 + num1;
            num1 = num2;
            num2 = num3;
            contador = contador + 1;

        }
    }
    public static void main(String[] args){

        int N = 21;

        Fibonacci(N);
    }
}

File: exercicio2.py
def Fibonacci(entrada):
    contador, numero_atual, proximo_numero = 0, 0, 1
    
    while contador < entrada:        
        print(f'{numero_atual}, ', end='')
        
        # Verifica se o número de entrada faz parte da sequência de fibonacci.
        if entrada == numero_atual:
            print(f'\nNúmero {numero_atual} pertence a sequência de fibonacci!')
            break
        elif numero_atual > entrada:
            print(f'\nO Numero {entrada} não pertence a sequência de fibonacci')
            break
        
        soma = numero_atual + proximo_numero
        numero_atual = proximo_numero
        proximo_numero = soma
        contador += 1      

      
entrada = int(input('Informe um número para verificar se o mesmo pertence a sequência de fibonacci.\n'))
Fibonacci(entrada)